src/main/java/dev/railroadide/railroad/settings/Settings.java

```diff
@@ -81,5 +85,18 @@ public class Settings {
             .defaultValue(new HashMap<>())
             .build());
 
+    public static final Setting<Map<String, List<KeybindData>>> KEYBINDS = registerSetting(Setting.builder((Class<Map<String, List<KeybindData>>>) (Class<?>) Map.class)
```

Comment:
```suggestion
    public static final Setting<Map<String, List<KeybindData>>> KEYBINDS = registerSetting(Setting.builder(new TypeToken<Map<String, List<KeybindData>>>() {})
```

You should be able to use a `TypeToken` here? 
`new TypeToken<Map<String, List<KeybindData>>>() {}`

Reply:
Nope, requires a class and I can't seem to get it working

Reply:
that shouldn't be the case?
https://github.com/Railroad-Team/RailroadCore/blob/7399a13a06c38c6727d0be8c17e800127e6702ea/src/main/java/dev/railroadide/core/settings/Setting.java#L156-L159